members/SNJG8473b29e.yaml

```diff
@@ -76,6 +76,14 @@ contact_form:
           values: ["$PHONE"]
           selectors: ["input[name='Phone_H']"]
           commands: ["elements[0].value = values[0].replace(/\D+/g,'').length > 10 ? values[0].replace(/\D+/g,'').substr(1) : values[0].replace(/\D+/g,'')" ]
+        - name: add-period-to-firstName
+          values: ["$NAME_FIRST"]
+          selectors: ["#FIRSTNAME"]
+          commands: ["elements[0].value = !elements[0].value || elements[0].value.length > 1 ? elements[0].value : elements[0].value + '.'" ]
```

Comment:
what is this doing?

Reply:
this (and the other js step below) add a period to this input if there's only one character present. This is to sort of "hack" the form validation of Gov Murphy that requires `first` and `last` names to have at least two characters. Luckily, the form doesn't care if some of those characters are not letters.

Reply:
if `!elements[0].value` is true then `elements[0].value` is set to `elements[0].value`, is that intended?

Reply:
Yeah, I think that's intended

cc @tirish

Reply:
Yeah, intended